worker-analytics-engine-forwarder/src/index.js

```diff
@@ -0,0 +1,137 @@
+/*
+  Using this template:
+  In wrangler.toml
+    * Edit the worker name.
+    * Edit the "dataset" field in the analytics_engine binding to specify the name of the Workers Analytics Engine dataset to write to.
+  In a terminal
+    * Run "wrangler secret put BEARER_TOKEN" and enter a secret value. You must also supply this token when calling the worker (see README.md).
+  In this file:
+    * Edit the "writeDataPoint" call in the "processLogEntry" function to map your event data to fields in the analytics engine data point.
+    * Optionally you could add additional code in that function to filter or process your messages before logging them.
+*/
+
+// processLogEntry maps incoming log/event lines onto an Analytics Engine datapoint and writes the datapoint to Analytics Engine.
+// As an example we've shown a mapping of the Cloudflare default HTTP requests log fields.
+// Textual fields are mapped to blobs and numeric fields to doubles.
+// Edit this to map the fields that are present in your data.
+function processLogEntry(ANALYTICS, data) {
+	ANALYTICS.writeDataPoint({
+		indexes: [data.ClientRequestHost || ''], // Supply one index
+		blobs: [
+			// Supply a maximum of 20 blobs (max total data size 5120 bytes)
+			data.RayID || '',
+			data.ClientIP || '',
+			data.ClientRequestMethod || '',
+			data.ClientRequestURI || '',
+			data.EdgeStartTimestamp || '',
+			data.EdgeEndTimestamp || '',
+			data.EdgeResponseStatus || '',
```

Comment:
Why not a double? Easier to do comparisons like EdgeResponseStatus > 400 (for example)